packages/magento-product-configurable/components/ConfigurableProductUrls/ConfigurableProductUrls.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from 'next/router'
+import { useEffect } from 'react'
+import { useConfigurableOptionsSelection } from '../../hooks/useConfigurableOptionsSelection'
+
+type ConfigurableProductUrlsProps = {
+  url_key?: string | null
+  index?: number
+}
+
+export function ConfigurableProductUrls(props: ConfigurableProductUrlsProps) {
+  const { url_key, index = 0 } = props
+  const router = useRouter()
+  const { configured } = useConfigurableOptionsSelection({ url_key, index })
+
+  useEffect(() => {
+    const currentUrlPath = router.asPath.split('/')
+    const currentUrlKey = currentUrlPath[currentUrlPath.length - 1]
+    const simpleProductUrlKey = configured?.configurable_product_options_selection?.variant?.url_key
```

Comment:
How do we determine if the simple is visible individually? It can be that some products aren't visible in the Catalog, right?